openff/toolkit/tests/test_forcefield.py

```diff
@@ -3242,7 +3242,10 @@ def check_molecule_constraints(cls, molecule, system, bond_elements, bond_length
                 molecule.atoms[atom2_idx].element.symbol,
             }
             assert atom_elements == bond_elements
-            assert np.isclose(distance / unit.angstrom, bond_length / unit.angstrom)
+            assert np.isclose(
+                distance.value_in_unit(unit.angstrom),
+                bond_length.value_in_unit(unit.angstrom),
+            )
```

Comment:
Out of curiosity, does the below work? I know pint understands units.

```python
assert np.isclose(distance, bond_length)
```

**Pint**

```python
>>> import pint
>>> unit = pint.UnitRegistry()
>>> a = np.array([1.1]) * unit.angstrom
>>> b = np.array([0.11]) * unit.nanometer
>>> np.isclose(a, b)
array([ True])
>>> a = 1.1 * unit.angstrom
>>> b = 0.11 * unit.nanometer
>>> np.isclose(a, b)
True
```

Reply:
No to both. :(

Perhaps something to keep in mind if/when the move to pint happens.

```python
>>> from openff.toolkit.topology.molecule import unit
>>> a = np.array([1.1]) * unit.angstrom
>>> b = np.array([0.11]) * unit.nanometer
>>> np.isclose(a, b)
array([False])
>>> a = 1.1 * unit.angstrom
>>> b = 0.11 * unit.nanometer
>>> np.isclose(a, b)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "<__array_function__ internals>", line 5, in isclose
  File "/Users/lily/anaconda3/envs/polymetrizer/lib/python3.9/site-packages/numpy/core/numeric.py", line 2355, in isclose
    xfin = isfinite(x)
TypeError: ufunc 'isfinite' not supported for the input types, and the inputs could not be safely coerced to any supported types according to the casting rule ''safe''
```

Reply:
This confusing behavior is one reason I use `pint` wherever I have the choice.